guide/portuguese/java/loops/index.md

```diff
@@ -1,24 +1,24 @@
 ---
 title: Loops
-localeTitle: rotações
+localeTitle: Loops
 ---
-# rotações
+# Loops
 
-Sempre que você precisar executar um bloco de código várias vezes, um loop geralmente vem a calhar.
+Sempre que você precisar executar um bloco de código várias vezes, um loop(ou laço) geralmente vem a calhar.
```

Comment:
I suggest to remove the parentheses or the word "ou".

```suggestion
Sempre que você precisar executar um bloco de código várias vezes, um loop(ou laço) geralmente vem a calhar.
```

Reply:
I made the change on the user's behalf.